hudi-hadoop-mr/src/test/java/org/apache/hudi/hadoop/TestHoodieParquetInputFormat.java

```diff
@@ -241,6 +242,38 @@ public void testIncrementalSimple() throws IOException {
         "We should exclude commit 100 when returning incremental pull with start commit time as 100");
   }
 
+  @Test
+  public void testMultiPartitionTableIncrematal() throws IOException {
```

Comment:
Incremental

Reply:
@leesf   thanks for your review.  already updated the code